### DISCUSSION TITLE
New Release for DataGrip 2024 compatibility too ?

### DISCUSSION THREAD

**Discussion by aadrian** [ng-galien/idea-plpgdebugger#259] [General]
Hi,

Any chance creating a new release and include compatibility for DataGrip 2024.x too?

Thank you.

**Comment by amerryma** [comment]
Hi, I'm also interested in this. I saw a branch called 242 which is what I would need. Is there any reason why this is not released yet? @ng-galien

**Comment by ng-galien** [comment]
Hi, there are breaking changes from Intellij. I've time to address it these next days

**Reply by pmonster** [reply]
How's it coming along - need any assistance?

**Reply by ng-galien** [reply]
Hi,  
The new version compatible with intellij 251 is released, waiting for approval from JB for marketplace availability.
